javalib/src/main/scala/java/lang/StackTrace.scala

```diff
@@ -105,8 +105,7 @@ private[lang] object StackTrace {
   private def normalizedLinesToStackTrace(
       lines: js.Array[String]): Array[StackTraceElement] = {
 
-    val NormalizedFrameLine = """^([^\@]*)\@(.*):([0-9]+)$""".re
-    val NormalizedFrameLineWithColumn = """^([^\@]*)\@(.*):([0-9]+):([0-9]+)$""".re
+    val NormalizedFrameLine = """^([^@]*)@(.*?):([0-9]+)(?::([0-9]+))?$""".re
 
     @inline def parseInt(s: String): Int =
       js.Dynamic.global.parseInt(s).asInstanceOf[Int]
```

Comment:
Somewhat unrelated, but shouldn't this set the radix explicitly?

Reply:
Not in this case. The strings come straight for a regex that only takes `[0-9]+`, so that will always be interpreted in base 10.

Reply:
Hah, I ran into this trap: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#octal_interpretations_with_no_radix